quantaq/client.py

```diff
@@ -45,7 +45,8 @@ def __init__(
 
         self._users = None
         self._devices = None
-        self._teams = None
+        self._orgs = None
```

Comment:
This is fine if only internal, though could consider switching to `_organizations` to stay self-consistent